bin/reslug_manual_section

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env ruby
+
+require File.expand_path("../../config/environment", __FILE__)
+require "manual_section_reslugger"
+require "logger"
+
+def usage
+  $stderr.puts %Q{
+USAGE: #{File.basename(__FILE__)} (manual_slug, old_manual_section_slug, new_manual_section_slug)
+
+manual_slug:
+  slug of manual (eg 'guidance/countryside-stewardship-manual')
+
+old_manual_section_slug:
+  current slug to be renamed (eg '8-terms-and-conditions')
+
+new_manual_section_slug:
+  current slug to be renamed (eg '8-scheme-requirements-and-procedures')
```

Comment:
This is the same description as `old_manual_section_slug` - maybe something like "new slug for the section" would be clearer?